libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/assign.pass.cpp

```diff
@@ -0,0 +1,61 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
```

Comment:
Mark the following tests as `XFAIL` on AIX:

```
libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/fetch_add.pass.cpp
libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/fetch_sub.pass.cpp
libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/operator.minus_equals.pass.cpp
libcxx/test/std/atomics/atomics.types.generic/atomics.types.float/operator.plus_equals.pass.cpp
```

You can use

```
// It looks like basic arithmetic operations on atomics don't work properly on AIX
// XFAIL: LIBCXX-AIX-FIXME
```

CC @daltenty